classes/classes/BodyParts/BodyPart.as

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.CoC_Settings;
+import classes.Creature;
+import classes.internals.SimpleJsonable;
+
+public class BodyPart extends SimpleJsonable {
+	include "../../../includes/appearanceDefs.as";
```

Comment:
BTDT with PlayerHelper.as
btw: I'm considering to swap Player.as and PlayerHelper.as, so I have access to the 'score' methods and player.race as well

Should be a matter of Renaming Player.as to PlayerBase.as and PlayerHelper.as to Player.as and then just fixing the `extends`